app/assets/javascripts/components/authentication/login_form.es6.jsx

```diff
@@ -24,15 +25,15 @@ class LoginForm extends Component {
   }
 
   createSession(event) {
-    Requester.post(
-      RouteConstants.users.login,
-      {
-        user: {
-          email: this.state.email,
-          password: this.state.password,
-        },
-      }
-    );
+    var params = {
+      user: {
+        email: this.state.email,
+        password: this.state.password,
+      },
+    };
+    var resolve = (response) => { window.location = RouteConstants.students.index };
+    var reject = (response) => { this.setState({ errors: true }) };
```

Comment:
We're going to be introducing something called the "rocket", which is a nice way of creating functions without having to use '.bind(this)' (I mentioned this earlier and here it is!).